check-diff.sh

```diff
@@ -589,6 +589,25 @@ function run_phpunit_travisci {
 		cd "$INSTALL_PATH"
 
 		echo "Location: $INSTALL_PATH"
+	elif [ "$PROJECT_TYPE" == theme ]; then
+		INSTALL_PATH="$WP_CORE_DIR/src/wp-content/themes/$PROJECT_SLUG"
+
+		# Rsync the files into the right location
+		mkdir -p "$INSTALL_PATH"
+		rsync -a $(verbose_arg) --exclude .git/hooks --delete "$PROJECT_DIR/" "$INSTALL_PATH/"
+		cd "$INSTALL_PATH"
+
+		# Clone the theme dependencies (i.e. plugins) into the plugins directory
+		if [ ! -z "$THEME_DEPS" ]; then
+			IFS=',' read -r -a dependencies <<< "$THEME_DEPS"
+			for dep in "${dependencies[@]}"
+			do
+				filename=$(basename "$dep")
+				git clone "$dep" "$WP_CORE_DIR/src/wp-content/plugins/${filename%.*}"
```

Comment:
Would `svn export` be an alternative? I'm using it in https://github.com/GlotPress/gp-translation-propagation/blob/1-unit-tests/.dev-lib for #157.